src/helpers/expense.helpers.js

```diff
@@ -0,0 +1,78 @@
+const userService = require('../services/user.service');
+const expenseService = require('../services/expense.service');
+
+const isUserExist = async (userId, res) => {
+  if (!(await userService.getById(userId))) {
+    res.status(400).send('User not found');
+
+    return true;
+  }
```

Comment:
Helpers should know anything about the response and what status/message to send. Just check if the user exists

Fix logic in all helpers